docuploader/__main__.py

```diff
@@ -89,12 +94,15 @@ def upload(
 
     docuploader.log.info("Loading up your metadata.")
     try:
-        shutil.copy(metadata_path, pathlib.Path(documentation_path) / "docs.metadata")
+        shutil.copy(metadata_path, pathlib.Path(documentation_path) / metadata_extension)
```

Comment:
Reuse `metadata_path`.

Reply:
I suppose this actually gets reduced to
```
shutil.copy(metadata_path, metadata_path)
```
I suppose the original intention was to transform other docs.metadata file names to `docs.metadata`, should I revert it to the original then?